portal.go

```diff
@@ -1299,6 +1296,11 @@ func (portal *Portal) sendMessage(intent *appservice.IntentAPI, eventType event.
 	if err != nil {
 		return nil, err
 	}
+
+	if intent.IsCustomPuppet {
+		wrappedContent.Raw = map[string]interface{}{doublePuppetKey: doublePuppetValue}
+	}
```

Comment:
This needs to override the raw map even when it's not a custom puppet

```suggestion
	} else {
		wrappedContent.Raw = nil
	}
```